examples/knowledge_tuning/instructlab/document_pre_processing.py

```
@@ -0,0 +1,75 @@
+# ### Create Seed Examples
+
+# %%
+from utils import DocProcessor
+
+output_dir = f"sdg_demo_output/"
+# This is where your PDFs are stored
+data_dir = '../document_collection/ibm-annual-report' 
+# It also have your QNA yaml file
+dp = DocProcessor(data_dir, user_config_path=f'{data_dir}/qna.yaml')
+
+### Using docling v1 json
```

Flake8 (7.2.0)

[error] too many leading '#' for block comment

(E266)